test/enable-disable-test.js

```diff
@@ -51,9 +52,51 @@ test('When disabled the circuit should always be closed', t => {
       breaker.fire(-1)
         .catch(e => t.equals(e, 'Error: -1 is < 0'))
         .then(() => {
-          t.ok(breaker.opened, 'should be closed');
+          t.ok(breaker.opened, 'should be open');
```

Comment:
Was this a typo? If yes, I took the liberty of correcting it :)

Reply:
🤣 most likely.  damn copy/paste